### DISCUSSION TITLE
Inconsistent spacing in Git status

### DISCUSSION THREAD

**Discussion by vvvetroff** [JanDeDobbeleer/oh-my-posh#5065] [Troubleshoot]
Currently, I am tweaking/configuring the catppuccin mocha theme to make it look like p10k. I only use like 4 git commands (status, add, commit, push), so it would be nice to see what was modified/untracked, what was added, and what has been committed, just like p10k does. 

My problem is that there is inconsistent spacing with ">" (sideways chevron) char when the dir is on a git branch or not. Image below:

<img width="670" alt="Screenshot 2024-06-12 at 11 30 18 PM" src="https://github.com/JanDeDobbeleer/oh-my-posh/assets/100363698/8fb02f66-8bb4-4e78-a623-4b91a6100751">

When I add spaces in between "{{.Working}}" and "{{.Staging}}" inside my toml config, it is fine when there is an untracked and a staged commit, but looks jarring without them. Image below: 

<img width="670" alt="Screenshot 2024-06-12 at 11 28 51 PM" src="https://github.com/JanDeDobbeleer/oh-my-posh/assets/100363698/9d56b202-7588-4af7-8dcb-ecc78ba0b018">

What changes can I make to my theme config to make the spacing consistent?

Here is my template code for reference:
`[[blocks.segments]]`
`type = 'git'`
`style = 'plain'`
`foreground = 'p:lavender'`
`template = '{{ .HEAD }} {{ if .Working }}{{ .Working }}{{ end }}{{ if .Staging }}{{ .Staging }}{{ end }}'`

**Comment by JanDeDobbeleer** [comment]
@vvvetroff can you share the full configuration?

**Reply by vvvetroff** [reply]
Sorry for late response, here it is. nvim makes this file not indented, if thats a problem.

[catppuccintheme.txt](https://github.com/user-attachments/files/15831270/catppuccintheme.txt)
